Onboarding: update screenshots (#27511)
home-assistant/home-assistant.github.io: source/getting-started/onboarding.markdown

Alright, you made it here. The tough part is done.

> After entering the Home Assistant devices address in the address bar of your browser, the preparation screen is shown. Depending on your hardware, preparation may take a while.

Reviewer: ```suggestion
After entering the Home Assistant device's address in your browser's address bar, the preparation screen is shown. Depending on your hardware, preparation may take a while.
```

 
1. Enter a name, username, and password.  Select **Create account**.

    ![Set your username and password.](/images/getting-started/username.png)

1. Enter a name for your home and define the location specific settings and the language of the user interface. 
   * To automatically populate these settings, select **Detect**.
   * If you'd rather not send your location, you can set these values manually.

    ![Define your location specific settings.](/images/getting-started/onboarding_location.png)
    
1. Select which information you are willing to share. 
    * Sharing is disabled by default. However, we would like to encourage you to share some of this data. 
    * This information helps us to find out which platforms we need to support and where to focus our efforts.
    * The data is anonymized and aggregated. To see the charts we generate out of this data, take a look at our [analytics page](https://analytics.home-assistant.io/). 
    
   ![Share anonymized data](/images/getting-started/onboarding_share_anonymized_info.png) 

> 1. Once you are done, click **Next**. 

Author: ```suggestion
1. Once you are done, select **Next**. 
```

>     * Home Assistant will then show any {% term devices %} that it has discovered on your network. 

Reviewer: ```suggestion
    * Home Assistant will then show any {% term devices %} it has discovered on your network.
```

    * Don't be alarmed if you see fewer items than shown below; you can always manually add devices later.

    ![Discovery of devices on your network.](/images/getting-started/onboarding_devices.png)

> 1. Finally, click **Finish**. 

Author: ```suggestion
1. Finally, select **Finish**. 
```
